.buildkite/commands/run-unit-tests.sh

```diff
@@ -20,13 +20,16 @@ bundle exec fastlane test_without_building name:UnitTests
 TESTS_EXIT_STATUS=$?
 set -e
 
-if [[ "$TESTS_EXIT_STATUS" -ne 0 ]]; then
-  # Keep the (otherwise collapsed) current "Testing" section open in Buildkite logs on error. See https://buildkite.com/docs/pipelines/managing-log-output#collapsing-output
-  echo "^^^ +++"
-  echo "Unit Tests failed!"
-fi
-
 echo "--- 📦 Zipping test results"
 cd fastlane/test_output/ && zip -rq WooCommerce.xcresult.zip WooCommerce.xcresult && cd -
 
+echo "--- 🚦 Report Tests Status"
+if [[ $TESTS_EXIT_STATUS -eq 0 ]]; then
+  echo "Unit Tests seems to have passed (exit code 0). All good 👍"
+else
+  echo "The Unit Tests, ran during the '🧪 Testing' step above, have failed."
+  echo "For more details about the failed tests, check the Buildkite annotation, the logs under the '🧪 Testing' section and the \`.xcresult\` and test reports in Buildkite artifacts."
```

Comment:
This `'🧪 Testing' section` part of message (and the similar one in the `run-ui-tests.sh` script above) may be a little bit misleading, because that's not the label of [the test step](https://github.com/woocommerce/woocommerce-ios/blob/192c84b4d74c138b8f3c519b323e3bd6e6e4da34/.buildkite/pipeline.yml#L81). I don't think we need to mention the section name in this message, because this message would show up in the said section of the buildkite build? Also, you can use `BUILDKITE_LABEL` env var if you want to include the step label in the message.

Reply:
Ah ok, yeah it was a blatant copy-paste from the WP/JPiOS message, I can remove the reference as it only appears on the logs (not the actual annotation). Updated in https://github.com/woocommerce/woocommerce-ios/pull/9726/commits/e061d6729317f6873dd33161ba75a0070b6df87f

Reply:
> expand the "~~~ 🧪 Testing" collapsible section in that log (which [this code previously auto-expanded for you in case of test failures](https://github.com/woocommerce/woocommerce-ios/commit/f2f9c3a2f23ba5477fb63f4736017f42b69c4010#diff-bd954ba2ef49cba9cd9950b5197be8e22f9c95f5482f03d0186eaf9404cda0e8L34-L38), and I'm not sure why we removed that trick in this PR?)

my bad, this was due to the copy-paste from WP/JPiOS, i'll open another PR to fix that. i think the simplified wording now leaves no room for misunderstandings, but i can update them for better consistency between this and WP/JPiOS.